### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Yazino12**
### In this pull:

- Initialize the project with components
- Add components: BookStore, BookList, Book, Navbar, AddBook
- Add pages: Categories
- Install dependencies: react-router-dom, UUID
- Use React hooks: state, setState
- Use React props and state lifting
- Add basic styling
- Update readme

**Commit #2:**
- Install prop-types to add Typechecking with Proptypes
- Enable Eslint

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/prop-types */
```
Comment (Deepakdanger) [inline src/components/AddBook.js:1]: - [ ] Kindly enable this ESlint and add prop-types to fix that linter issue. Please refer [this](https://reactjs.org/docs/typechecking-with-proptypes.html) for understanding prop-type.

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/prop-types */
```
Comment (Deepakdanger) [inline src/components/Book.js:1]: - [ ] Kindly enable this ESlint and add prop-types to fix that linter issue. Please refer [this](https://reactjs.org/docs/typechecking-with-proptypes.html) for understanding prop-type.

File: src/components/BookList.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/prop-types */
```
Comment (Deepakdanger) [inline src/components/BookList.js:1]: - [ ] Kindly enable this ESlint and add prop-types to fix that linter issue. Please refer [this](https://reactjs.org/docs/typechecking-with-proptypes.html) for understanding prop-type.

File: src/components/BookStore.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/prop-types */
```
Comment (Deepakdanger) [inline src/components/BookStore.js:1]: - [ ] Kindly enable this ESlint and add prop-types to fix that linter issue. Please refer [this](https://reactjs.org/docs/typechecking-with-proptypes.html) for understanding prop-type.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Yazino12, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- Good PR :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Yazino12 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Github inline comments feature is used properly.
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookStore.js (hunk 4 above)
Comment (Yazino12) [reply src/components/BookStore.js]: Thanks, @Deepakdanger for the guidance, I will apply these changes right away 😊.

File: src/components/BookList.js (hunk 3 above)
Comment (Yazino12) [reply src/components/BookList.js]: Thanks, @Deepakdanger for the guidance, I will apply these changes right away 😊.

File: src/components/Book.js (hunk 2 above)
Comment (Yazino12) [reply src/components/Book.js]: Thanks, @Deepakdanger for the guidance, I will apply these changes right away 😊.

File: src/components/AddBook.js (hunk 1 above)
Comment (Yazino12) [reply src/components/AddBook.js]: Thanks, @Deepakdanger for the guidance, I will apply these changes right away 😊.

**Pull request merged**